app/services/complete_moab_service/create.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 module CompleteMoabService
-  # Creates CompletedMoab and associated objects based on a moab on disk.
+  # Creates CompletedMoab and associated objects based on a moab on storage.
```

Comment:
change this comment in other similar classes -- maybe do a search on "disk" in codebase  (missed in create_after_validation)